addons/hr/models/res_users.py

```diff
@@ -160,6 +160,15 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False, submenu
                 result['toolbar']['action'] = [act for act in result['toolbar']['action'] if act['id'] != change_password_action.id]
         return result
 
+    def get_employee_values_sync(self, vals):
```

Comment:
Use private method, please

```suggestion
    def _get_employee_values_sync(self, vals):
```